### PR TITLE
CHE-6845: Disable breakpoints

### PR DESCRIPTION
**Pull request by tolusha**
### What does this PR do?
Ability to disable breakpoints

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/6845

Ability to disable breakpoints

#### Release Notes

Ability to disable breakpoints

screenshot from 2017-11-06 13-31-37
screenshot from 2017-11-06 13-31-20

### CODE REVIEW

File: plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/actions/DeleteBreakpointAction.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+  @Override
+  public void actionPerformed(ActionEvent e) {
+    Breakpoint breakpoint = (Breakpoint) getTemplatePresentation().getClientProperty(BREAKPOINT);
+    breakpointManager.delete(breakpoint);
+  }
+
+  @Override
+  public void updateInPerspective(ActionEvent event) {}
```
Comment (AndrienkoAleksandr) [inline plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/actions/DeleteBreakpointAction.java:52]: Do we need some checks here?

File: plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/BreakpointItemRender.java [hunk 2]
```diff
@@ -14,46 +14,65 @@
...
+
     TableCellElement label = Elements.createTDElement();
 
     SafeHtmlBuilder sb = new SafeHtmlBuilder();
     // Add icon
     sb.appendHtmlConstant("<table><tr><td>");
-    SVGResource icon = debuggerResources.breakpoint();
-    if (icon != null) {
-      sb.appendHtmlConstant("<img src=\"" + icon.getSafeUri().asString() + "\">");
+    sb.appendHtmlConstant("<div class=\"");
```
Comment (AndrienkoAleksandr) [inline plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/debug/BreakpointItemRender.java:46]: I think we can try: 
```
    elemental.html.DivElement divElement = Elements.createDivElement("class1", "class2");
    divElement.setId("some-id");
```

File: plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/actions/DeleteBreakpointAction.java (hunk 1 above)
Comment (tolusha) [reply plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/actions/DeleteBreakpointAction.java:52]: No checks here

**Comment by tolusha** [comment]
```
[TEST] Command line: /home/tolusha/projects/che/selenium/che-selenium-test/selenium-tests.sh --test=org.eclipse.che.selenium.debugger.** --skip-sources-validation
[TEST]
[TEST] Local results:
[TEST] 	- Run: 	     46
[TEST] 	- Failed:     5
[TEST]
[TEST] REGRESSION (5):
[TEST] 	org.eclipse.che.selenium.debugger.CppProjectDebuggingTest.shouldDebugCppProject
[TEST] 	org.eclipse.che.selenium.debugger.CppProjectDebuggingTest.stopDebug
[TEST] 	org.eclipse.che.selenium.debugger.DebugExternalClassTest.shouldDebugJreClass
[TEST] 	org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.shouldDebugMethodLocalInnerClass
[TEST] 	org.eclipse.che.selenium.debugger.InnerClassAndLambdaDebuggingTest.stopDebug
```

**Comment by tolusha** [comment]
ci-build

**Review by AndrienkoAleksandr — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Review by vzhukovs — APPROVED** [review]

**Pull request merged**